pkg/server/server.go

```diff
@@ -17,14 +17,14 @@ import (
 	"github.com/go-kit/log/level"
 	"github.com/gorilla/mux"
 	"github.com/grafana/ckit/memconn"
+	_ "github.com/grafana/pyroscope-go/godeltaprof/http/pprof" // anonymous import to get the godeltaprof handler registered
```

Comment:
Should this also be imported somewhere in the Flow mode binary?

Reply:
🤷 sure !